6-controls.js

```diff
@@ -0,0 +1,98 @@
+// Take a number of rows and output a right angled triangle
+var star = "";
+var printTriangle = function (rows) {
+    for (var i = 0; i < rows; i++) {
+        console.log(star += "*");
+        for (var i = 0; i <= rows; i++) {
+            console.log(star += "*");
+        }
+    }
+}
+
+printTriangle(3);
+
+
+// Take a number of rows and output a equalateral triangle
+var equalateralTriangle = function(size){
+    var stars = '';
+    var spaces = '';
+    for(var i = 0 ; i < size; i++){
+        spaces = '';
+        for(var j = size -1-i; j > 0; j--) {
+            spaces += ' ';
+        }
+        stars += '*';
+        if(stars.length > 1) {
+            stars += '*'
+        }
+        console.log(spaces+stars);
+    }
+};
+
+equalateralTriangle(10);
+
+
+//Output the Fibonacci sequence up, up to a certain value.
+var fibonacciSequence = function(endNumber) {
+    var a = 0, b = 1, f = 1, count;
+    var fibArray = [a, b];
+    for (count = 3; count <= endNumber; count++) {
+        f = a + b;
+        fibArray.push(f);
+        a = b;
+        b = f;
+    }
+    return fibArray;
+};
+
+console.log(fibonacciSequence(10));
+
+//Looping through numbers
+var loopThroughNumbers = function(maxNumber) {
```

Comment:
This is a valid solution ti the FizzBuzz problem ( Google it). There are several workable solutions, which is why I set it. 
I prefer:
````
if divisible by 3 and 5 {
  output FIZZBUZZ
}
else if divisible by 3  {
  output FIZZ
}
else if divisible by 5  {
  output BUZZ
}
else {
    output number
}
````